microsoft_intune/help.md

```diff
@@ -56,6 +56,39 @@ Example input:
 
 ### Actions
 
+#### Add application to a policy
+
+This action is used to add application to a policy.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|application_name|string|None|True|Application name|None|Adobe Reader|
+|device_name|string|None|True|Device name|None|None|
+|device_type|string|None|True|Device type|['Android', 'iOS']|None|
+|policy_name|string|None|True|Policy name|None|my-android-app-policy|
+
+Example input:
+
+```
+{
+  "application_name": "Adobe Reader",
+  "policy_name": "my-android-app-policy"
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|success|object|False|Return true if action was successfully performed on device|
+
+Example output:
+
+```
```

Comment:
Need JSON formatted example output from a real request. Sanitize any data that shouldn't be viewable publicly